SECU74010-project.csproj

```diff
@@ -6,6 +6,19 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <RootNamespace>SECU74010-project</RootNamespace>
     <AssemblyName>$(AssemblyName.Replace(' ', '_'))</AssemblyName>
+    <PublishWithAspNetCoreTargetManifest>false</PublishWithAspNetCoreTargetManifest>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
would this mean the tests are accessible when its running or just adding test as a target state?

Reply:
Oh good catch. TBH I think that `<publish...` line is not needed.

We can run tests with `dotnet test`, but I tried my best to prevent the tests from being bundled with a release build. But I should test that further